Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.Tests/Lib/ReadFileTestBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NUnit.Framework;
+using Frends.SFTP.ReadFile.Definitions;
+using Frends.SFTP.ReadFile.Enums;
+
+namespace Frends.SFTP.ReadFile.Tests;
+public class ReadFileTestBase
+{
+    protected static Connection _connection;
+    protected static Input _input;
+    protected static string _content;
```

Comment:
All of the properties should be `internal` instead of `protected`, since they are not constants and they are not used in this class but in the classes which inherit this class.